sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/_evaluators/_multimodal/_content_safety_multimodal.py

```diff
@@ -0,0 +1,244 @@
+                        for content in message["content"]:
+                            if content.get("type") == "image_url":
+                                image_url = content.get("image_url")
+                                if image_url and 'url' in image_url:
+                                    image_found = True
+                                    
+                    if isinstance(message["content"], dict):
+                        msg = f"Content in each turn must be a string or array. Message number: {msg_num}"
+                        raise EvaluationException(
+                            message=msg,
+                            internal_message=msg,
+                            target=ErrorTarget.CONTENT_SAFETY_MULTIMODAL_EVALUATOR,
+                            category=ErrorCategory.INVALID_VALUE,
+                            blame=ErrorBlame.USER_ERROR,
+                        )         
+            else:
+                try:
+                    from azure.ai.inference.models import ChatRequestMessage, UserMessage, AssistantMessage, SystemMessage, ImageContentItem
+                except ImportError:
+                    error_message = "Please install 'azure-ai-inference' package to use SystemMessage, AssistantMessage"
+                    raise MissingRequiredPackage(message=error_message)
+                else:
+                    if isinstance(messages[0], ChatRequestMessage):
+                        if not isinstance(message, UserMessage) and not isinstance(message, AssistantMessage) and not isinstance(message, SystemMessage):
+                            msg = f"Messsage in array must be a strongly typed class of [UserMessage, SystemMessage, AssistantMessage]. Message number: {msg_num}"
+                            raise EvaluationException(
+                                message=msg,
+                                internal_message=msg,
+                                target=ErrorTarget.CONTENT_SAFETY_MULTIMODAL_EVALUATOR,
+                                category=ErrorCategory.INVALID_VALUE,
+                                blame=ErrorBlame.USER_ERROR,
+                            )  
+                        if message.content and isinstance(message.content, list):
+                            image_items = [item for item in message.content if isinstance(item, ImageContentItem)]
+                            if len(image_items) > 0:
+                                image_found = True
+        if image_found is False:    
+            msg = f"Message needs to have multimodal input like images."
+            raise EvaluationException(
+                message=msg,
+                internal_message=msg,
+                target=ErrorTarget.CONTENT_SAFETY_MULTIMODAL_EVALUATOR,
+                category=ErrorCategory.INVALID_VALUE,
+                blame=ErrorBlame.USER_ERROR,
+            )
+
+    def _get_harm_severity_level(self, harm_score: float) -> Union[HarmSeverityLevel, float]:
```

Comment:
let's make this a utility function since it's not using `self`

Reply:
Hmm, It looks like a dup. we are already have same function in utils. I will remove it.